CompetitionMars/CompetitionMars/Pages/CertificationPage.cs

```diff
@@ -0,0 +1,258 @@
+            
+            List<Certification> certificationTestData = jsonHelperObject.ReadCertificateTestDataFromJson(jsonFilePath);
+            // List<Certification> certificationTestData = JsonConvert.DeserializeObject<List<Certification>>(jsonData);
+            string result = "";
+
+            foreach(var certificate in certificationTestData)
+            {
+                string certificateName = certificate.Certificate;
+                string certificateFrom = certificate.From;
+
+                IReadOnlyCollection<IWebElement> rows = driver.FindElements(By.XPath("//th[text()='Certificate']//ancestor::thead//following-sibling::tbody/tr"));
+
+                foreach (IWebElement row in rows)
+                {
+                    IWebElement certificateNameElement = row.FindElement(By.XPath("./td[1]"));
+                    IWebElement certificateFromElement = row.FindElement(By.XPath("./td[2]"));
+
+                    string deletedCertificateName = certificateNameElement.Text;
+                    string deletedCertificateFrom = certificateFromElement.Text;
+
+                    if ((deletedCertificateName != certificate.Certificate) && (deletedCertificateFrom != certificate.From))
+                      {
+                        result = "Deleted";
+                        break;
+                      }
+                    else
+                    {
+                        result = "Not Deleted";
+                    }
+                    
+
+                }
+            }
+           
+            return result;
+            //  if (country.Equals(Country, StringComparison.OrdinalIgnoreCase) && collegeName.Equals(CollegeName, StringComparison.OrdinalIgnoreCase) && title.Equals(Title, StringComparison.OrdinalIgnoreCase) && degree.Equals(Degree, StringComparison.OrdinalIgnoreCase) && year.Equals(Year, StringComparison.OrdinalIgnoreCase))
+
+        }
+
+        public void AddEmptyCertificationField(string Certificate, string From, string CertificationYear)
+        {
+            certificationTab.Click();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+
+            addNewButton.Click();
+
+            certificateTextbox.SendKeys(Certificate);
+            certificateFromTextbox.SendKeys(From);
+            certificateYearDropdown.SendKeys(CertificationYear);
+
+            addButton.Click();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+        }
+
+        public string GetEmptyFieldErrorMessage()
+        {
+            IWebElement actualErrorMessage = driver.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+            return actualErrorMessage.Text;
+        }
+
+        public void AddSameCertificationSameYear(string Certificate,string From, string CertificationYear)
+        {
+            certificationTab.Click();
+            addNewButton.Click();
+
+            certificateTextbox.SendKeys(Certificate);
```

Comment:
if you look at the three methods about you are doing the same things e.g send keys you could have a separate method which the other methods can use!